indico/modules/rb_new/client/js/modules/admin/LocationPage.jsx

```diff
@@ -0,0 +1,178 @@
+                   format={formatters.trim} formatOnBlur
+                   label={Translate.string('Room name format')}
+                   validate={val => {
+                       const rv = v.required(val);
+                       if (rv) {
+                           return rv;
+                       }
+                       const missing = ['{building}', '{floor}', '{number}'].filter(x => !val.includes(x));
+                       if (missing.length) {
+                           return PluralTranslate.string(
+                               'Missing placeholder: {placeholders}',
+                               'Missing placeholders: {placeholders}',
+                               missing.length,
+                               {placeholders: missing.join(', ')}
+                           );
+                       }
+                   }} />
+            <Field name="map_url_template" component={ReduxFormField} as="input"
+                   format={formatters.trim} formatOnBlur
+                   label={Translate.string('Map URL template')}
+                   validate={val => {
+                       if (val && !val.match(/https?:\/\/.+/)) {
+                           return Translate.string('Please provide a valid URL');
+                       }
+                   }}>
+                <p className="field-description">
+                    <Translate>
+                        Indico can show a link to an external map when a room is associated to an event.
+                        Specify the template to generate those links, using any of the following placeholders:
+                        {' '}
+                        <Param name="placeholders" wrapper={<code />}
+                               value="{id}, {building}, {floor}, {number}, {lat}, {lng}" />
+                    </Translate>
+                </p>
+            </Field>
+        </>
+    );
+
+    renderDeleteMessage = ({name}) => {
+        return (
+            <Translate>
+                Are you sure you want to delete the location
+                {' '}
+                <Param name="name" wrapper={<strong />} value={name} />
+                {' '}
+                and all data associated with it?
+            </Translate>
+        );
+    };
+
+
+    render() {
+        const {
+            isFetching, locations,
+            actions: {createLocation, updateLocation, deleteLocation},
+        } = this.props;
+
+        return (
+            <EditableList title={Translate.string('Locations')}
+                          addModalTitle={Translate.string('Add location')}
+                          isFetching={isFetching}
+                          items={locations}
+                          initialEditValues={snakifyKeys}
+                          initialAddValues={{
+                              room_name_format: '{building}/{floor}-{number}',
+                              map_url_template: 'https://www.google.com/maps/place/{lat},{lng}',
```

Comment:
Do we want to promote google maps? :trollface:
This could also be a placeholder

Reply:
Placeholder wouldn't be great for the user since he'd have to type the URL. Having a link to google maps is the most convenient option for someone who doesn't want to bother setting up their own map etc. But we could also use OSM if they have a public site similar to google maps.

Reply:
- OSM: https://www.openstreetmap.org/?mlat=46.23227762&mlon=6.05491571&zoom=18
- Google: https://www.google.com/maps/place/46.23227762,.05491571

Reply:
+1 for OSM

Reply:
One benefit of linking to Google would be that people who want to use it for GPS navigation are already in the right place.

Reply:
Problem solved, now there's no default and you can choose between google, osm and a custom url